nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -287,6 +295,11 @@ in {
           message = "Please specify exactly one of adminpass or adminpassFile";
         }
       ];
+
+      warnings = optional (cfg.poolConfig != null) ''
```

Comment:
@averelld your intention with this was to present the user with a second warning to clarify the first warning I assume?